src/menus/picklist/picklist.ts

```diff
@@ -37,11 +38,35 @@ export class NglPicklist {
   @Input() open: boolean;
   @Output() openChange = new EventEmitter();
 
+  @Input() filter: string | Function;
+  @Input() filterPlaceholder = 'Filter Options';
```

Comment:
I would prefer to initialize this to empty string, as Select2 does. In the future (unless you want to do this now) we will allow users to configure this through `provideNgConfig`.